components/benefit_sponsors/app/views/benefit_sponsors/profiles/employers/employer_profiles/_sponsored_benefit.html.erb

```diff
@@ -131,9 +131,13 @@
           <div class="col-xs-4">
             <div class="row">
               <span>Employer Estimated Maximum Monthly Cost</span>
-              <% estimator = ::BenefitSponsors::Services::SponsoredBenefitCostEstimationService.new %>
-	            <% estimate = estimator.calculate_estimates_for_benefit_display(sponsored_benefit) %>
-              <h2><%= number_to_currency(estimate[:estimated_sponsor_exposure]) %></h2>
+              <%if sponsored_benefit.benefit_package.benefit_application.aasm_state == :canceled %>
```

Comment:
@ymhari can we use the actual method instead of checking the aasm_state? `.canceled?`